src/components/DocumentList/DocumentList.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { ReactElement } from "react";
+import { ScrollView } from "react-native";
+import { DocumentListItem } from "./DocumentListItem";
+
+export interface DocumentItem {
+  id: string;
+  title: string;
+  isVerified?: boolean;
+  lastVerification?: number;
+}
+
+export interface DocumentList {
+  documents: DocumentItem[];
+  navigateToDoc: (documentId: string) => void;
+}
+
+export const DocumentList = ({
+  documents,
+  navigateToDoc
+}: DocumentList): ReactElement => {
+  const renderedDocumentListItem = documents.map(
+    (doc): ReactElement => (
+      <DocumentListItem
+        key={doc.id}
+        title={doc.title}
+        isVerified={doc.isVerified}
+        lastVerification={doc.lastVerification}
+        onPress={(): void => navigateToDoc(doc.id)}
```

Comment:
The return type of functions can also be removed here too. Don't want to be defining the return type of every anonymous function. Let's add 
```
"@typescript-eslint/explicit-function-return-type": [
        "warn",
        {
          "allowExpressions": true,
          "allowTypedFunctionExpressions": true
        }
      ],
```
to our eslint too. (https://github.com/Open-Attestation/react-component-template/blob/master/package.json#L130)